Taskfile.yml

```diff
@@ -88,9 +88,12 @@ tasks:
 
   cov:
     desc: Generate coverage
+    env:
+      # https://github.com/golang/go/issues/65570
+      GOEXPERIMENT: nocoverageredesign
```

Comment:
@0xDEC0DE seem reasonable ?

Reply:
Well, isn't that fucking lovely...

Yes, this seems to be the best location on the turd sandwich to take a bite.